ui/src/views/infra/HostAdd.vue

```diff
@@ -94,9 +94,8 @@
         <a-input :placeholder="placeholder.username" v-model="username"></a-input>
       </div>
 
-      <div class="form__item required-field" v-if="selectedClusterHyperVisorType !== 'VMware'">
-        <div class="form__label"><span class="required">* </span>{{ $t('label.password') }}</div>
-        <span class="required required-label">{{ $t('label.required') }}</span>
+      <div class="form__item" v-if="selectedClusterHyperVisorType !== 'VMware'">
+        <div class="form__label">{{ $t('label.password') }}</div>
```

Comment:
Maybe be have it not required just for kvm? (for ex. required for xenserver?)

Reply:
ping @weizhouapache when you've time - why not put explicit checks for example a radio/button or toggle to switch between password or ssh-key auth just for KVM (however for XenServer too this feature can be done).

Reply:
@rhtyd 
good idea. will make the UI change.
let's support kvm at first, then xenserver (this is not needed for vmware).